index.html

```diff
@@ -104,7 +105,6 @@ <h2>Profesional Art Printing Data</h2>
                       <li>Ruby</li>
                   </ul>
               </div>
-                <button type="button">See project</button>
             </div>
             <div class="showcase">
                 <div class="showcase-image">
```

Comment:
- [ ] As seen from the Figma Design. I kindly suggest that you ensure that:

- When the project is hovered on with the mouse, the description and dark opacity should disappear and a See Project button appears to the bottom of the image and take the whole image width 👍🏿

- When the project is hovered over, it should look exactly like the first project you have, while when your image is not hovered over, it should look like the rest of the projects.

<img width="718" alt="image" src="https://github.com/iankiema/My-Portifolio-Personal/assets/52934128/d259ddca-1a3f-4c6d-ad17-74955d773fa0">